clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/download/CopybookDownloaderForUssTest.spec.ts

```diff
@@ -18,14 +18,15 @@ import {
   allUSSFilemembers,
   getUSSContentsMock,
 } from "../../../../__mocks__/getZoweExplorerMock.utility";
-import * as vscode from "vscode";
+import * as vscode from "../../../../__mocks__/vscode";
```

Comment:
Why the change? This should be handled transparently by jest.

Reply:
Fixed